### DISCUSSION TITLE
課題3についての質問②

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#12] [Q&A]
「５０５０」と「２５」の結果を出力することはできたのですが、for文内の数字箇所を順番に打ち込んでいき、結果を導き出しました。そのため、「○○だから、ここに1を入れる」といった考え方ができないでいます。
なぜ、そうなるのか教えていただけますと幸いです。

**Comment by nsc-kiwamu** [comment]
朝会で直接聞きながら話した方が良さそうですね。
